packages/core/src/Provider/Provider.ts

```diff
@@ -1,4 +1,5 @@
 import { Percent } from '../Cardano';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This is curious. Seems like this lint rule is a bit naive

Reply:
Yeah, it doesn't know that it's just a type. If we start running into more of those we could loosen it up by allowing devDependencies in the rule

Reply:
Next update might resolve this: https://github.com/import-js/eslint-plugin-import/blob/main/CHANGELOG.md#unreleased